assets/performanceprofile/tuned/openshift-node-performance

```diff
@@ -24,7 +24,46 @@ isolated_cores={{.IsolatedCpus}}
 {{end}}
 
 not_isolated_cores_expanded=${f:cpulist_invert:${isolated_cores_expanded}}
-automatic_pstate=${f:intel_recommended_pstate}
+
+# Intel specific tuning options
+{{if ${f:lscpu_check:"^Vendor ID:\s*(.*)$"} == "GenuineIntel"}}
```

Comment:
Eventually we will have to deal with sysctls as well. I think separating those to a sub-profile will work better. The main profile will then just include (if we can do that for variables). The current logic here is already becoming hard to read.

Reply:
If I am understanding you correctly, then you think it might be better to look at a more file orientated approach, where we might have separate files that get included when appropriate, such as:

- openshift-node-performance-shared (with all common configuration)
- openshift-node-performance-intel-x86
- openshift-node-performance-amd-x86
- openshift-node-performance-arm-aarch64

And then we could overlay a specific platform profile on top of the shared profile to come up with a final profile?

Reply:
We already have shared, that is the main profile. And we already have -rt for real time changes. But the others might make the whole thing more readable once it is more than just few kernel args.

Reply:
I've updated the draft PR to use separate files as described above with the include tuned directive